tango/device_proxy.py

```diff
@@ -334,29 +339,35 @@ def __DeviceProxy__setattr(self, name, value):
 
     try:
         self.__refresh_cmd_cache()
-    except:
-        pass
+    except Exception as e:
+        if cause is None:
+            cause = e
 
     if name_l in self.__get_cmd_cache():
         raise TypeError('Cannot set the value of a command')
```

Comment:
Shouldn't this line include `cause`, similar to line 332?

Reply:
Good catch. Thanks!